vhdbuilder/packer/install-dependencies.sh

```diff
@@ -345,6 +345,8 @@ if grep -q "fullgpu" <<< "$FEATURE_FLAGS" && grep -q "gpudaemon" <<< "$FEATURE_F
 fi
 fi
 
+systemctlEnableAndStart cgroup-stats.timer || true
```

Comment:
```suggestion
systemctlEnableAndStart cgroup-stats.timer || exit 1
systemctlEnableAndStart cgroup-stats.service || exit 1
```

also let's exit on failure